### PR TITLE
Refactor core/BackgroundJobs

### PR DESCRIPTION
**Pull request by shdehnavi**
## Summary
The required adjustments have been made to the classes in `/core/BackgroundJobs ` namespace.

The improvements:

- Adding return types

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

### CODE REVIEW

File: core/BackgroundJobs/BackgroundCleanupUpdaterBackupsJob.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
 	/**
 	 * This job cleans up all backups except the latest 3 from the updaters backup directory
 	 */
-	public function run($arguments) {
+	public function run($arguments): void {
```
[the remark this answers is not in the document]
Comment (shdehnavi) [reply core/BackgroundJobs/BackgroundCleanupUpdaterBackupsJob.php]: It's addressed in the next commit

[the remark this answers is not in the document]
Comment (shdehnavi) [reply core/BackgroundJobs/BackgroundCleanupUpdaterBackupsJob.php]: It's addressed in the next commit

File: core/BackgroundJobs/LookupServerSendCheckBackgroundJob.php [hunk 2]
```diff
@@ -40,7 +40,7 @@
 		parent::__construct($time);
 	}
 
-	public function run($arguments) {
+	public function run($arguments): void {
```
[the remark this answers is not in the document]
Comment (shdehnavi) [reply core/BackgroundJobs/LookupServerSendCheckBackgroundJob.php]: It's addressed in the next commit

[the remark this answers is not in the document]
Comment (shdehnavi) [reply core/BackgroundJobs/LookupServerSendCheckBackgroundJob.php]: It's addressed in the next commit

**Review by ChristophWurst — APPROVED** [review]

**Review by come-nc — APPROVED** [review]

**Pull request merged**